tools/ci/after_install.sh

```diff
@@ -12,3 +12,25 @@ if [ "$TEST_SUITE" = "spec:javascript" ]; then
   yarn list
   echo "travis_fold:end:YARN_LOCK"
 fi
+
+MASTERHASH=`git rev-parse master`
+PRESENT=`git show "$MASTERHASH" | wc -l`
+
+if [ "$TEST_SUITE" = "spec" -a "$TRAVIS_PULL_REQUEST" -a "$PRESENT" != 0]; then
```

Comment:
How could $PRESENT ever not be 0?

Reply:
It's also not the number of commits (as is presented below)...this would just be the number of lines in the commit message + the boilerplate lines